src/chain/network_event.rs

```diff
@@ -80,6 +80,9 @@ pub enum AccumulatingEvent {
 
     // Prune the gossip graph.
     ParsecPrune,
+
+    // Opaque user-defined event.
+    Custom(Vec<u8>),
```

Comment:
A better name was suggested for this event, so I'll be making this change:
```suggestion
    User(Vec<u8>),
```